### PR TITLE
FEATURE: change layout of escaped_fragment_ topic page to table one l…

### PR DESCRIPTION
**Pull request by mrfinch**
…ike live discourse

- also removes post links
- adds created on since last activity doesn't make much sense on archived site

I tested generated HTML at https://search.google.com/structured-data/testing-tool/u/0/ and it detects the same content as before.

This is a part of https://meta.discourse.org/t/improving-discourse-static-html-archive/112497, I will be making incremental changes for the same instead of creating one big PR.

Adding screenshot for the new layout. cc @coding-horror  let me know if you have any feedback for it.

Screenshot from 2019-03-25 22-44-40

### CODE REVIEW

File: app/views/list/list.erb [hunk 1]
```diff
@@ -38,28 +38,52 @@
...
+                <p class='excerpt'>
+                  <%= t.excerpt.html_safe %>
+                </p>
+              <% end %>
+            </td>
+            <td>
+              <span class='posts' title='<%= t 'posts' %>'><%= t.posts_count %></span>
+            </td>
+            <td>
+              <%= t.created_at.strftime("%B %e %Y") %>
```
Comment (gschlager) [inline app/views/list/list.erb]: You should use a localized date format.

```suggestion
              <%= l t.created_at, format: :date_only %>
```

File: app/views/list/list.erb [hunk 2]
```diff
@@ -38,28 +38,52 @@
...
-      <% if (!@category || @category.has_children?) && t.category %>
-        <span class='category'>[<a href='<%= t.category.url %>'><%= t.category.name %></a>]</span>
-      <% end %>
-      <span class='posts' title='<%= t 'posts' %>'>(<%= t.posts_count %>)</span>
+<div itemscope itemtype='http://schema.org/ItemList'>
+  <table class='topic-list'>
+
+    <thead>
+      <tr>
+        <th>Topic</th>
```
Comment (gschlager) [inline app/views/list/list.erb]: These strings should use translations.

```suggestion
        <th><%= t 'js.topic.title' %></th>
```

File: app/views/list/list.erb [hunk 3]
```diff
@@ -38,28 +38,52 @@
...
+        <th>Replies</th>
```
Comment (gschlager) [inline app/views/list/list.erb]:
```suggestion
        <th><%= t 'js.replies' %></th>
```

File: app/views/list/list.erb [hunk 4]
```diff
@@ -38,28 +38,52 @@
...
+        <th>Created On</th>
```
Comment (gschlager) [inline app/views/list/list.erb]:
```suggestion
        <th><%= t 'js.created' %></th>
```

**Comment by mrfinch** [comment]
@gschlager, I have made the changes.

**Comment by ZogStriP** [comment]
Thanks 👍

**Pull request merged**